pkgs/by-name/sw/sway/package.nix

```diff
@@ -31,8 +31,10 @@ let
        exec ${lib.optionalString dbusSupport "${dbus}/bin/dbus-run-session"} ${lib.getExe sway} "$@"
      fi
    '';
-in symlinkJoin {
-  name = "${sway.meta.mainProgram}-${sway.version}";
+in symlinkJoin rec {
+  pname = lib.replaceStrings ["-unwrapped"] [""] sway.pname;
```

Comment:
nit: not sure if just setting "sway" as a literal string would be simpler

Reply:
then i would have to do the same for swayfx

Reply:
then please ignore my nit :)